EnglishTranslation-master/ETC.tsv

```diff
@@ -6335,67 +6335,67 @@ ETC_20150317_006335	Salavija Hill
 ETC_20150317_006336	Abandoned Camp
 ETC_20150317_006337	Hollow Tree Forest
 ETC_20150317_006338	Field of Reeds
-ETC_20150317_006339	Ishiblysh Square
+ETC_20150317_006339	Isblyskes Square
 ETC_20150317_006340	Ascetic Sanctuary
 ETC_20150317_006341	Nevasingas Highway
-ETC_20150317_006342	Kabeti Hill
+ETC_20150317_006342	Kabantis Hill
 ETC_20150317_006343	Nomads Camp
-ETC_20150317_006344	Zakole Shore
+ETC_20150317_006344	Pusmenulio Shore
 ETC_20150317_006345	Pilgrim's Tomb
-ETC_20150317_006346	Pasukti Path
-ETC_20150317_006347	Noreti Fountain
+ETC_20150317_006346	Plunksnos Path
+ETC_20150317_006347	Noru Fountain
 ETC_20150317_006348	Rudiziu Trail
 ETC_20150317_006349	Vinas Brewery
-ETC_20150317_006350	Oshimas Highway
+ETC_20150317_006350	Oshimo Highway
 ETC_20150317_006351	Chliz Uphill
 ETC_20150317_006352	Branginti Hill
-ETC_20150317_006353	Pamaish Hill
+ETC_20150317_006353	Sukurio Hill
 ETC_20150317_006354	Saint's Tomb
 ETC_20150317_006355	Sausas Fork
 ETC_20150317_006356	Bridge of Farewell
 ETC_20150317_006357	Hermit's House
 ETC_20150317_006358	Erienos Chapel
-ETC_20150317_006359	Malone Trail
+ETC_20150317_006359	Malones Trail
 ETC_20150317_006360	Sestine Wilderness
 ETC_20150317_006361	Peddler Camp
-ETC_20150317_006362	Niekas Field
+ETC_20150317_006362	Nieko Field
 ETC_20150317_006363	Meeting Plaza
 ETC_20150317_006364	Wedroka Access Road
 ETC_20150317_006365	Mirshta Lake
 ETC_20150317_006366	Meile Oratorium
 ETC_20150317_006367	Pamaldu Groom
 ETC_20150317_006368	Collapsed Dome
-ETC_20150317_006369	Arka Chapel
+ETC_20150317_006369	Arkos Chapel
 ETC_20150317_006370	Reception Room
-ETC_20150317_006371	Veinika Altar
-ETC_20150317_006372	Uolie Chapel
+ETC_20150317_006371	Karunos Altar
+ETC_20150317_006372	Uolos Chapel
 ETC_20150317_006373	Altar of Stability
 ETC_20150317_006374	Small Hall
-ETC_20150317_006375	Liudesy Highway
-ETC_20150317_006376	Olivinas Vacant Lot
+ETC_20150317_006375	Liudesio Highway
+ETC_20150317_006376	Olivino Vacant Lot
 ETC_20150317_006377	Fasadu Rest Area
-ETC_20150317_006378	Laukti Antenave
-ETC_20150317_006379	Atejima Concourse
+ETC_20150317_006378	Laukimo Antenave
+ETC_20150317_006379	Atejimo Concourse
 ETC_20150317_006380	Small Reception Room
-ETC_20150317_006381	Pasara Altar
-ETC_20150317_006382	Tikintie Gallery
-ETC_20150317_006383	Ishgaki Rest Area
+ETC_20150317_006381	Pasalos Altar
+ETC_20150317_006382	Tikinciu Gallery
+ETC_20150317_006383	Aptvertas Rest Area
```

Comment:
Not sure about this one. Wiki gives me "Stone Wall", and "Aptvertas" means "enclosed", so it kind of makes sense, but at the same time, this is not a condition but a name, so it should indicate wall rather than enclosement. I think. 

So maybe... Siena Rest Area (wall rest area) ?